warehouse/redshift/redshift.go

```diff
@@ -332,6 +332,7 @@ func (rs *HandleT) loadTable(tableName string, tableSchemaInUpload, tableSchemaA
 	sanitisedSQLStmt, regexErr := misc.ReplaceMultiRegex(sqlStatement, map[string]string{
 		"ACCESS_KEY_ID '[^']*'":     "ACCESS_KEY_ID '***'",
 		"SECRET_ACCESS_KEY '[^']*'": "SECRET_ACCESS_KEY '***'",
+		"SESSION_TOKEN '[^']*'":     "SESSION_TOKEN '***'",
```

Comment:
session token will be active only for a while like an hour. That could be reason why it was omitted

Reply:
Let's remove it because it is sensitive. wdyt?

Reply:
`When SESSION_TOKEN is specified, you must also use ACCESS_KEY_ID and SECRET_ACCESS_KEY to provide temporary access key credentials`
Not required. wdyt?

Reply:
Adding this in logs also doesn't make it productive. Since it is customer specific tokens which are sensitive information it's better to not log it.